packages/search-ui/src/__tests__/actions/setFilter.test.ts

```diff
@@ -1,29 +1,30 @@
-import { setupDriver } from "../../test/helpers";
+import { setupDriver, SubjectArguments } from "../../test/helpers";
 import {
   itResetsCurrent,
   itFetchesResults,
   itUpdatesURLState
 } from "../../test/sharedTests";
 
 // We mock this so no state is actually written to the URL
-jest.mock("../../URLManager.js");
+jest.mock("../../URLManager");
 import URLManager from "../../URLManager";
+const MockedURLManager = jest.mocked(URLManager, true);
 
 beforeEach(() => {
-  URLManager.mockClear();
+  MockedURLManager.mockClear();
 });
 
 describe("#setFilter", () => {
   function subject(
-    name,
-    value,
-    type,
+    name?,
+    value?,
+    type?,
     {
```

Comment:
Got it. Never seen this before, good to know. I thought since all the parameters are `any`, this includes `undefined`, but apparently that doesn't include the parameter being skipped entirely.